frontend/tests/playwright/product-list/filters.spec.ts

```diff
@@ -5,12 +5,21 @@ import { waitForAlgoliaSearch, resolvePath } from '../utils';
 // Check that the refinement value is in the current refinements.
 async function checkRefinementValue(
    value: string | null | undefined,
-   page: Page
+   page: Page,
+   layout: 'desktop' | 'mobile'
 ) {
    if (!value) {
       throw new Error('Could not find ' + value);
    }
-   await expect(page.getByTestId(`current-refinement-${value}`)).toBeVisible();
+   if (layout === 'desktop') {
+      await expect(
+         page.getByTestId(`current-refinement-${value}`).first()
+      ).toBeVisible();
+   } else {
+      await expect(
+         page.getByTestId(`current-refinement-${value}`).last()
+      ).toBeVisible();
+   }
```

Comment:
It's not clear to me why we're making the distinction between mobile and desktop in this test. When I inspect the `<CurrentRefinements>` component, it appears that all current refinements are rendered on both mobile and desktop, so I'm confused why we can't make the same assertion on both.

Reply:
We have multiple `<CurrentRefinements>` components in dom. However their visibility depends on the layout.
If we are on a mobile layout `<CurrentRefinements>` is visible only in the toolbar component.
If we are on a desktop layout `<CurrentRefinements>` is located in the upper part of the side panel.

To avoid altering the test, I made the above distinction.

Reply:
Thanks for the explanation! I missed the fact that there are 2 components.